salt/profiles/init.sls

```diff
@@ -28,6 +28,7 @@ profiles-install:
         - user: {{ pillar.elife.deploy_user.username }}
         - require:
             - profiles-repository
+            - profiles-app-config
```

Comment:
install.sh needs `profiles-app-config`? Should only require the `requirements.txt` file for the virtualenv setup?

Reply:
It'll need it if things like https://github.com/elifesciences/profiles/pull/10/files#diff-3fbb47e318cd8802bd325e7da9aaabe8R15 happen.

Reply:
ok, but I would change the order of states in the `.sls` file to keep them in sequence

Reply:
Done.